### ISSUE TITLE
Remove SetUnderrunCallback

### ISSUE THREAD

**Issue by hajimehoshi** [hajimehoshi/oto#48]
I think @faiface is using this, but I was wondering if this is really needed now.

**Comment by faiface** [comment]
Yeah, I'm using it and I think it probably is needed for some debugging purposes. Like, when you're hitting a lot of underruns, it's useful to know when they're happening to fix it.

**Comment by hajimehoshi** [comment]
If underruns happens, probably is the game itself heavy?

**Comment by faiface** [comment]
Yes, exactly. But it may happen under some condition in the game and not in others. Knowing when it happens can be very useful and that's exactly what `SetUnderrunCallback` can be used for.

**Comment by faiface** [comment]
Well, I don't know. Maybe it is useless, but I have some vague memory of actually using it back when developing Beep. But, if you really want to, it's probably OK to remove it.

**Comment by hajimehoshi** [comment]
OK so my proposal is to remove this once, and let's revisit when we think we really need this.
